src/main/java/net/atos/client/bag/BAGClientService.java

```diff
@@ -99,7 +99,7 @@ public AdresIOHal readAdres(final URI adresURI) {
 
     private Invocation.Builder createInvocationBuilder(final URI uri) {
         return JAXRSClientFactory.getOrCreateClient().target(uri)
-                .request("application/hal+json", "application/problem+json")
-                .header(X_API_KEY, API_KEY);
+                                 .request("application/hal+json", "application/problem+json")
```

Comment:
Wow, quite some indent. Oh well, I guess this is the standard then right?

Reply:
Not sure I like these huge indents. Actually I don't. ;-)

Reply:
That can be changed to an ident of 8 spaces if we agree on that.

Reply:
Done. @edgarvonk have a look at this variant

At least it's easily configurable via IntelliJ :)